style.css

```diff
@@ -0,0 +1,727 @@
+/* stylelint-disable no-descending-specificity */
+* {
+  margin: 0;
+}
+
+.main {
+  background-color: #e5e5e5;
+}
+
+header {
+  padding-top: 26px;
+  padding-left: 5%;
+  padding-right: 5%;
+  height: 72px;
+  background: #3e3c3c;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+a:hover {
+  text-decoration: underline;
+  color: #f55800;
+}
+
+.nav-right ul {
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
+
+.nav-right ul li {
+  margin-left: 32px;
+}
+
+.icon-letter {
+  width: 22px;
+  height: 18px;
+  border: 2px solid #67798e;
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.intro-page {
+  overflow: hidden;
+  display: grid;
+  grid-template-columns: 10% 50% 40%;
+  align-items: center;
+  justify-items: center;
+  background: #1c1a19;
+  height: 870px;
+  width: 100%;
+}
+
+.intro-social-icon {
+  grid-column: 1/2;
+}
+
+.fa-envelope-square {
+  width: 22px;
+  height: 18px;
+}
+
+.intro-social-icon ul {
+  list-style: none;
+}
+
+.intro-social-icon ul li {
+  margin-bottom: 20px;
+}
+
+.icon {
+  color: #dbd8d7;
+  text-decoration: none;
+}
+
+.intro-text {
+  grid-column: 2/3;
+  margin-left: 50px;
+  z-index: 99;
+  padding-right: 12px;
+}
+
+.intro-text h1 {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 72px;
+  line-height: 81px;
+  color: #f55800;
+}
+
+.intro-text h2 {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.intro-page p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 32px;
+  color: #f4f5f7;
+}
+
+.background-img {
+  grid-column: 3/4;
+  overflow: hidden;
+  height: 700px;
+  width: 650px;
+  background-image: url("/Portfolio-Desktop/img/Group.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+@media only screen and (max-width: 900px) {
+  .intro-page {
+    height: 100%;
+    width: 100%;
+  }
+
+  .background-img {
+    height: 200px;
+    width: 250px;
+    position: absolute;
+    left: 65%;
+    right: -30.09%;
+    top: 28%;
+    bottom: -13.63%;
+  }
+}
+
+@media only screen and (max-width: 600px) {
```

Comment:
Please use the value for min-width and max-width property in range of 992px - 1100px.